CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Cromwell Change Log
 
+## 80 Release Notes
+
+### Direct WES support in Cromwell
+
+Cromwell 80 will no longer support the wes2cromwell project within the Cromwell repository.
```

Comment:
Tense: as you write now, release 80 is in the future, but when users actually read the release note, it will be in the present

```suggestion
Cromwell 80 no longer supports the wes2cromwell project within the Cromwell repository.
```